### DISCUSSION TITLE
Can I change the variant "default" style from the theme settings?

### DISCUSSION THREAD

**Discussion by borachoidev** [mantinedev/mantine#4690] [Q&A] [answered]
Hi! Is there a way to globally change the button label color when the variant is set to default? 
I'd like the button with the default variant to have a style where the label color matches the theme's primary color. 
Could you please assist me with this :)

**Comment by rtivital** [comment, marked as answer]
You can override Button variants styles with Styles API on MantineProvider – https://mantine.dev/theming/mantine-provider/#variants-on-mantineprovider
